librepo/downloader.c

```diff
@@ -293,7 +293,15 @@ is_max_mirrors_unlimited(const LrDownload *download)
 }
 
 static gboolean
-can_try_more(const LrDownload *download, int num_of_tried_mirrors, gboolean complete_path_or_baseurl)
+/**
+ * @brief Can be tried another mirror or retried baseurl or fullpath
+ *
+ * @param download p_download:...
+ * @param num_of_tried_mirrors p_num_of_tried_mirrors:...
+ * @param complete_path_or_baseurl p_complete_path_or_baseurl: determine type of download - mirrors or baseurl/fullpath
+ * @return gboolean Return TRUE when another chance to download is allowed.
+ */
+can_try_more_paths(const LrDownload *download, int num_of_tried_mirrors, gboolean complete_path_or_baseurl)
```

Comment:
I don't think the `_path` suffix si good, how about naming the function `can_retry_download`?